core/src/test/scala/prometheus4cats/MetricsFactoryDslTest.scala

```diff
@@ -103,4 +105,13 @@ object MetricsFactoryDslTest {
   infoBuilder.contramap[List[(Label.Name, String)]](_.toMap)
   infoBuilder.build
   infoBuilder.resource
+
+  val doubleSummaryBuilder =
+    factory
+      .summary("summary")
+      .ofDouble
+      .help("some summary")
+      .quantile(Summary.QuantileDefinition(1.0, 0.1))
```

Comment:
`Summary.QuantileDefinition(1.0, 0.1)` feels a bit clunky, maybe this should be two separate types...

Reply:
Answering my own question: changed this to two separate refined types